src/js/features/tracks.js

```diff
@@ -168,7 +168,10 @@ Object.assign(MediaElementPlayer.prototype, {
 
       for (let i = 0; i < outEvents.length; i++) {
         player.captionsButton.addEventListener(outEvents[i], function () {
-          addClass(this.querySelector(`.${t.options.classPrefix}captions-selector`), `${t.options.classPrefix}offscreen`);
+          // add a small timeout so mobile devices can activate the buttons before focus is lost
+          setTimeout(() => {
+            addClass(this.querySelector(`.${t.options.classPrefix}captions-selector`), `${t.options.classPrefix}offscreen`);
+          }, 20);
```

Comment:
Why is there a timeout of 20ms? When there is a slow mobile device, that takes longer than 20ms, it would still break or not?

Reply:
On mobile devices the chapters/caption menu is hidden on "focusout". The timeout is merely to delay the hiding of the menu, so the touch can trigger the button the user is trying to press. This should work regardless of the devices speed, unless the whole device is locking up an the same touch handler that triggered the "focusout" hangs, instead of triggering the click/touch event on the button. Both events are triggered by the same touch.

Reply:
Changed timeout to 0 ms, because all it nereds to do is decouple the focusout-Handler.

Reply:
Thanks.